src/model_io/urdf/include/private/URDFParsingUtils.h

```diff
@@ -72,6 +72,14 @@ std::string inline intToString(const int inInt)
     return ss.str();
 }
 
+std::string inline intToString(const size_t inInt)
+{
+    std::stringstream ss;
+    ss << inInt;
+    return ss.str();
```

Comment:
There should be ``std::to_string``

Reply:
I prefer to always avoid the use of `std::to_string`, as on integers it works fine, but for floating points create critical problems. However this is just adding a function that already existed for a different type, we can cleanup the function all together.